src/Test.FunctionalTests.Sarif/v2/ConverterTestData/PREfast/TFS378661.cpp.xml

```diff
@@ -17,9 +17,15 @@
     <CATEGORY>
       <RULECATEGORY>mspft</RULECATEGORY>
     </CATEGORY>
+    <ADDITIONALINFO>
+      <entry key="SuppressedMatch" />
+      <entry key="HashKey:abcdefabcdefabcdefacbdeacbdefacbdefabcdadeca=" />
+      <entry key="SuppressJustification:Scheduled to fix in Milestone 32." />
+      <entry key="MatchingScore:45.4" />
+    </ADDITIONALINFO>
     <PATH>
       <SFA>
-        <FILEPATH>d:\src\sarif-sdk\src\sarif.functionaltests\convertertestdata\prefast\src\fixedbugs\</FILEPATH>
+        <FILEPATH>d:\src\sarif-sdk\src\sarif.functionaltests\convertertestdata\prefast\src\fixedbugs</FILEPATH>
```

Comment:
>fixedbugs [](start = 91, length = 9)

NIT: Since you made this change to demonstrate what happens when there's no trailing slash, consider changing the last component of the filename to something like `no-trailing-slash`.